docset/winserver2022-ps/addsdeployment/Install-ADDSDomain.md

```diff
@@ -216,15 +254,21 @@ Accept wildcard characters: False
 ```
 
 ### -InstallDns
-Indicates that the cmdlet installs and configures the DNS Server service for the domain or domain tree.
-For domain installation, if this parameter is left unspecified and the parent domain (or in the case of a domain tree, the forest root domain) already hosts and stores the DNS names for the domain, then the default for this parameter is $true and the DNS server is installed.
-Otherwise, if DNS domain names are hosted outside of Active Directory, the default is $false and no DNS server is installed.
 
-To test if DNS domain names are hosted outside of Active Directory, this cmdlet uses a start of authority (SOA) type DNS query.
-For instance, if the value of *NewDomainName* parameter is corp.contoso.com, Active Directory performs an SOA query for corp.contoso.com and ensures that the zone name in the response is corp.contoso.com.
+Indicates that the cmdlet installs and configures the DNS Server service for the domain or domain
+tree. For domain installation, if this parameter is left unspecified and the parent domain (or in
+the case of a domain tree, the forest root domain) already hosts and stores the DNS names for the
+domain, then the default for this parameter is $true and the DNS server is installed. Otherwise, if
+DNS domain names are hosted outside of Active Directory, the default is $false and no DNS server is
+installed.
+
+To test if DNS domain names are hosted outside of Active Directory, this cmdlet uses a start of
+authority (SOA) type DNS query. For instance, if the value of _NewDomainName_ parameter is
+corp.contoso.com, Active Directory performs an SOA query for corp.contoso.com and ensures that the
+zone name in the response is corp.contoso.com.
```

Comment:
```suggestion
Indicates that the cmdlet installs and configures the DNS Server service for the domain or domain
tree. For domain installation, if this parameter is left unspecified and the parent domain (or in
the case of a domain tree, the forest root domain) already hosts and stores the DNS names for the
domain, then the default for this parameter is `$true` and the DNS server is installed. Otherwise,
if DNS domain names are hosted outside of Active Directory, the default is `$false` and no DNS
server is installed.

To test if DNS domain names are hosted outside of Active Directory, this cmdlet uses a start of
authority (SOA) type DNS query. For instance, if the value of **NewDomainName** parameter is
`corp.contoso.com`, Active Directory performs an SOA query for `corp.contoso.com` and ensures that
the zone name in the response is `corp.contoso.com`.
```